src/renderer/__helpers__/jest.setup.env.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Jest Global Setup
+ * This runs once before all test suites
+ */
+export default () => {
+  // Sets timezone to UTC for consistent date/time in tests and snapshots
+  process.env.TZ = 'UTC';
```

Comment:
Prevent datetimes changing in snapshots when `pnpm test` is run in different timezones (ie: Austrlaia 🦘)